src/utils/upload/index.ts

```diff
@@ -35,6 +58,10 @@ export function areFilesSame(
      * precision of file times to prevent timing attacks and fingerprinting.
      * Context: https://developer.mozilla.org/en-US/docs/Web/API/File/lastModified#reduced_time_precision
      */
+    if (fileHashExists(existingFile) && fileHashExists(newFile)) {
+        return areFilesWithFileHashSame(existingFile, newFile);
```

Comment:
we should move this if block above the comment as that comment is related to the  second if block and not this one

Reply:
Ah yes, sorry